stories/Drawer.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
 import { Drawer } from '@textkernel/oneui';
 
 storiesOf('Atoms|Drawer', module)
     .addDecorator(withKnobs)
-    .add('Drawer', () => (
-        <Drawer
-            initialIsOpen
-            title={text('Drawer title', '12/13 processed (2 need review, 1 failed)')}
-        >
-            {text(
-                'Content',
-                `
+    .add(
+        'Drawer',
+        () => (
+            <Drawer
+                isShown={boolean('isShown', true)}
```

Comment:
can we add `initialIsExpanded` as a knob too?

Reply:
I don't see sense to add ```initialIsExpanded``` into knob. ```initialIsExpanded``` just impacts on initial render. Drawer doesn't reflect on this change  if we will change ```initialIsExpanded``` after render

Reply:
good point :-)